integration-tests/src/test_loop/builder.rs

```diff
@@ -43,11 +44,15 @@ pub struct TestLoopBuilder {
     test_loop: TestLoopV2,
     genesis: Option<Genesis>,
     clients: Vec<AccountId>,
+    /// Accounts whose clients should be configured as an archival node.
+    /// This should be a subset of the accounts in the `clients` list.
+    archival: HashSet<AccountId>,
```

Comment:
nit: `archival` -> `archival_clients` to have consistent naming with the previous field

Reply:
done.